src/globalStyles.tsx

```diff
@@ -1,13 +1,23 @@
 import { Global, css } from '@emotion/react';
 
-import theme from './theme';
+//To do: when we have moved away from Emotion, think about what we do with the file
 
 // loosely based on https://hankchizljaw.com/wrote/a-modern-css-reset + a height declaration on the next container...
 const globalStyles = (
   <Global
     styles={css`
       @import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&display=swap');
 
+      :root {
+        --blue-color: #0a165e;
```

Comment:
I only have one concern in all of this excellent goodness.

Reply:
Would there still be an advantage in referring to pallette variables by names which indicate their use  rather than their color?

e.g. "--primary-color" "--active-color" "--warning-color" "--success-color" ?

Reply:
agree with the above - if you'd like a reference point, here's how we approach this in Source:
- Palette and colour definition file: https://github.com/guardian/csnx/blob/main/libs/%40guardian/source-foundations/src/colour/palette.tsx#L384
- Storybook showing the palette: https://guardian.github.io/csnx/?path=/docs/source-foundations_palette--palette

The key difference is that there's a separation between the "palette" of available colours in the design system that we dip into to create a mapping. https://github.com/guardian/csnx/blob/main/libs/%40guardian/source-foundations/src/colour/palette.tsx#L4 

For our use here though, I'd be a proponent of skipping the "palette" data structure and just referencing the hex codes directly, even if there is repetition

Reply:
I know what you mean, I was thinking we could document them in Storybook under this is how we use our colours instead. I've found that in projects I end up having a random addition of colour variables anyway so it felt nice to have them all colours. 

On the flip side, if we ever wanted to update the styles of the site it is probably nicer if we had variable names that indicate use. I could ask Rachel if she had a list of palette variables she'd expect to reach for.

Reply:
I'm not particularly wedded to either way, happy to go with the majority on this.

Reply:
Ok, so Rachel and I have been having a chat. It doesn't seem there is a general consensus to this. Quick summary of the pros and cons of the approaches:

### Variable names that indicate use
#### Strengths 
- We could be very consistent in how we use the colour palette 
- Allows us the ability to quickly change up the look and feel of the site. 
#### Drawbacks
- Systems designed this way can be quite rigid 
- There could be an amount of up front work suggest by Rachel in mapping out the different states we may want or if we add variables as we go along this could lead to inconsistencies depending on the preference of different developers/designers.

### Colour variable names
#### Advantages
- The fairly small colour palette we have means a relatively small set of variables to use
- Allows designers and developers to combine colours in whatever way they wish without adding new variable names
#### Disadvantages
- Use of written documentation to indicate how they should be used
- Potential for colour combinations to be combined in an inaccessible way

I think after chatting with Rachel that colour variable names feel more flexible for an open source project which may have a variety of different people inputting into it, especially as it is a relatively small site with a fairly modest colour palette. I'd be interested in experimenting with just colour variables and seeing if that causes confusion or doesn't work when we grow the design system.

Reply:
Agreed, happy to run with the colour variable names for now - let's see how it plays out :)